helm/ph-ee-engine/templates/tests/integration-test.yaml

```diff
@@ -1,58 +1,70 @@
-          imagePullPolicy: "{{ .Values.integration_test.imagePullPolicy }}"
-          command: [ "/bin/bash" ]
-          args: [ "-c", '{{ .Values.integration_test.gradle.command }}' ]
-          env:
-            - name: "OPERATIONS-APP_CONTACTPOINT"
-              value: "{{ .Values.integration_test.operations_app.contactpoint }}"
-            - name: "BULK-PROCESSOR_CONTACTPOINT"
-              value: "{{ .Values.integration_test.bulk_processor.contactpoint }}"
-            - name: "CHANNEL-CONNECTOR_CONTACTPOINT"
-              value: "{{ .Values.integration_test.channel_connector.contactpoint }}"
-            - name: "KAFKA_BROKERS"
-              value: "{{ .Values.integration_test.kafka.brokers }}"
-            - name: "KAFKA_TOPICS"
-              value: "{{ .Values.integration_test.kafka.topics }}"
-            - name: "ZEEBE-OPERATIONS_CONTACTPOINT"
-              value: "{{ .Values.integration_test.zeebe_operations.contactpoint }}"
-            - name: "ZEEBE-OPERATIONS_NO-OF-WORKFLOWS"
-              value: "{{ .Values.integration_test.zeebe_operations.no_of_workflows }}"
-            - name: "MAX-RETRY-COUNT"
-              value: "{{ .Values.integration_test.max_retry_count }}"
-            - name: "RETRY-INTERVAL"
-              value: "{{ .Values.integration_test.retry_intervals }}"
-            - name: "DEFAULTS_TENANT"
-              value: "{{ .Values.integration_test.defaults.tenant }}"
-            - name: "DEFAULTS_AUTHORIZATION"
-              value: "{{ .Values.integration_test.defaults.authorization }}"
-            - name: "CHANNEL_BASE-URL"
-              value: "{{ .Values.integration_test.channel.base_url }}"
-            - name: "LOAN_BASE-URL"
-              value: "{{ .Values.integration_test.loan.base_url }}"
-            - name: "SAVINGS_BASE-URL"
-              value: "{{ .Values.integration_test.savings.base_url }}"
-          resources:
-            limits:
-              memory: "{{ .Values.integration_test.limits.memory }}"
-              cpu: "{{ .Values.integration_test.limits.cpu }}"
-            requests:
-              memory: "{{ .Values.integration_test.requests.memory }}"
-              cpu: "{{ .Values.integration_test.requests.cpu }}"
-      restartPolicy: Never
-  backoffLimit: 4
-
+  containers:
+  - name: integration-test
+    image: docker.io/openmf/ph-ee-integration-test:v1.3.0
+    imagePullPolicy: "Always"
+    command: [ "/bin/bash" , "-c" ]
+    args: 
+    # - ./gradlew test -Dcucumber.filter.tags="@bd012" ; echo 'Test complete' ; sleep 300 ; echo 'pod terminate'
+    - ./gradlew test -Dcucumber.filter.tags="@gov" ; echo 'Test complete' ; sleep 300 ; echo 'pod terminate'    
+    env:
+      - name: "OPERATIONS_APP_CONTACTPOINT"
+        value: "http://ph-ee-operations-app:80"
+      - name: "BULK_PROCESSOR_CONTACTPOINT"
+        value: "https://ph-ee-connector-bulk:80"
+      - name: "CHANNEL_CONNECTOR_CONTACTPOINT"
+        value: "https://ph-ee-connector-channel:80"
```

Comment:
connector channel is now running on port 8443. Please change the port 80 to 8443 here